static/app/views/issueDetails/traceTimeline/traceTimeline.spec.tsx

```diff
@@ -187,9 +189,29 @@ describe('TraceTimeline', () => {
       await screen.findByText('SELECT "sentry_monitorcheckin"."monitor_id"')
     ).toBeInTheDocument();
     expect(screen.queryByLabelText('Current Event')).not.toBeInTheDocument();
+
+    // Test analytics
     expect(useRouteAnalyticsParams).toHaveBeenCalledWith({
       trace_timeline_status: 'empty',
     });
+    // XXX: The component is being rendered twice, thus, two calls to trackAnalytics
+    expect(trackAnalytics).toHaveBeenCalledTimes(2);
```

Comment:
The parent component is not rendering twice - only the TraceIssueEvent one. I'm still investigating.